docs/docs/components/guidelines/README.md

```diff
@@ -239,7 +239,7 @@ Registry components are organized by app in the `components` directory of the
   `/components/twitter/sources/search-mentions/search-mentions.js`.
 
 You can explore examples in the [components
-directory](https://github.com/pipedreamhq/pipedream/tree/master/components).
```

Comment:
Broken link. Do you need a `/code` prefix here, too?

Reply:
Oops, my bad 😊 
Fixed!